### PR TITLE
{geo}[foss/2019b] WRF v4.1.3

### PR DESCRIPTION
**Pull request by tstrempel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/w/WRF/WRF-4.1.3-foss-2019b-dmpar.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    ('JasPer', '2.0.14'),
+    ('netCDF', '4.7.1'),
+    ('netCDF-Fortran', '4.5.2'),
+]
+
+# limit parallel build to 20
+maxparallel = 20
+
+buildtype = "dmpar"
+versionsuffix = '-%s' % buildtype
```
Comment (smoors) [inline easybuild/easyconfigs/w/WRF/WRF-4.1.3-foss-2019b-dmpar.eb:40]: @tstrempel can you please move these 2 lines to the top of the file (below version)?

Comment (smoors) [reply easybuild/easyconfigs/w/WRF/WRF-4.1.3-foss-2019b-dmpar.eb:40]: and while you're doing that, could you do the same for the intel version? WRF-4.1.3-intel-2019b-dmpar.eb

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 2 out of 3 (2 easyconfigs in this PR)
node253.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2650 v4 @ 2.20GHz (broadwell), Python 2.7.5
See https://gist.github.com/a70d600645fc4592a2934d92618d66ae for a full test report.

**Comment by tstrempel** [comment]
@smoors I have no way of testing with the intel toolchain locally (no IntelMPI), so I can't reproduce the error

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 1 out of 3 (2 easyconfigs in this PR)
node361.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/aa25ef7db26900f68c208d59626ee33e for a full test report.

**Comment by smoors** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
> @smoors I have no way of testing with the intel toolchain locally (no IntelMPI), so I can't reproduce the error

The issue is probably just a timeout during the installation, @smoors saw other installation failures due to a (very) slow filesystem in #10714...

We can bump the timeout a bit in the WRF easyblock, doesn't hurt...

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node3137.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/e2cd50f97e9dc683ee51528385f749bd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3413.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/927a1b353e5e1e64ae971d090ac44b23 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2448.golett.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/cb34f82acd2b5877f76771d68362384d for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
node254.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2683 v4 @ 2.10GHz (broadwell), Python 2.7.5
See https://gist.github.com/a65439502787ea273a362d3abf2eb136 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3101.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/cb5b1db76f7672670091e307fee02f1f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @tstrempel!

**Pull request merged**